ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -648,10 +648,12 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FILTERS_ADDITIONAL: ""
       FIPS_ENABLED: "true"
+      INDEX_IMAGE: quay.io/redhat-user-workloads/ocp-isc-tenant/security-profiles-operator-fbc-4-12:latest
       TEST_FILTERS: ~HyperShiftMGMT&;~DEPRECATED&
       TEST_PARALLEL: "3"
       TEST_SCENARIOS: Security_Profiles_Operator
     test:
+    - ref: security-profiles-konflux-catalogsource
```

Comment:
No need to create a new step. Please reuse the step [file-integrity-konflux-catalogsource](https://github.com/openshift/release/blob/master/ci-operator/step-registry/file-integrity/konflux-catalogsource/file-integrity-konflux-catalogsource-ref.yaml). Update the CATALOG_SOURCE_NAME env the ICSP to be used is enough.